game/views.py

```
@@ -4033,20 +4034,26 @@ def update_opening_stats(request):
             },
             status=400,
         )
 
-    accuracy = max(0, min(100, accuracy))
+    correct_move = max(0, correct_move)
+    incorrect_move = max(0, incorrect_move)
+
+    total_moves = correct_move + incorrect_move
+    accuracy = round((correct_move / total_moves) * 100, 2) if total_moves > 0 else 0
 
     progress, first_completion = update_opening_progress(
```

Ruff (0.16.0)

[warning] Unpacked variable `progress` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)